bigbluebutton-html5/imports/ui/components/error-screen/component.jsx

```diff
@@ -80,40 +78,59 @@ const propTypes = {
     PropTypes.string,
     PropTypes.number,
   ]),
+  error: PropTypes.object,
+  errorInfo: PropTypes.object,
 };
 
 const defaultProps = {
   code: '500',
   callback: () => {},
   endedReason: null,
+  error: {},
+  errorInfo: {},
 };
 
 class ErrorScreen extends PureComponent {
   componentDidMount() {
     const { code, callback, endedReason } = this.props;
-    const log = code === '403' ? 'warn' : 'error';
-    AudioManager.exitAudio();
+    // stop audio
+    document.querySelector('audio').pause();
+    navigator
+      .mediaDevices
+      .getUserMedia({ audio: true, video: true })
+      .then((m) => m.getTracks().forEach((t) => t.stop()));
     callback(endedReason, () => Meteor.disconnect());
-    logger[log]({ logCode: 'startup_client_usercouldnotlogin_error' }, `User could not log in HTML5, hit ${code}`);
+    console.error({ logCode: 'startup_client_usercouldnotlogin_error' }, `User could not log in HTML5, hit ${code}`);
   }
 
   render() {
     const {
       intl,
       code,
       children,
+      error,
+      errorInfo,
     } = this.props;
+    let formatedMessage = 'Oops, something went wrong';
+    let errorMessageDescription = Session.get('errorMessageDescription');
+    if (intl) {
+      formatedMessage = intl.formatMessage(intlMessages[defaultProps.code]);
 
-    let formatedMessage = intl.formatMessage(intlMessages[defaultProps.code]);
+      if (code in intlMessages) {
+        formatedMessage = intl.formatMessage(intlMessages[code]);
+      }
 
-    if (code in intlMessages) {
-      formatedMessage = intl.formatMessage(intlMessages[code]);
-    }
+      errorMessageDescription = Session.get('errorMessageDescription');
 
-    let errorMessageDescription = Session.get('errorMessageDescription');
+      if (errorMessageDescription in intlMessages) {
+        errorMessageDescription = intl.formatMessage(intlMessages[errorMessageDescription]);
+      }
+    }
 
-    if (errorMessageDescription in intlMessages) {
-      errorMessageDescription = intl.formatMessage(intlMessages[errorMessageDescription]);
+    if (error) {
+      errorMessageDescription = `Error: ${JSON.stringify(error)}
```

Comment:
Looks like this log may actually include newline characters, making it difficult to parse/consume. Please double check or just use spaces instead of newline.